tests/plugins/simple_text_ckeditor/test_plugin.py

```diff
@@ -0,0 +1,52 @@
+"""
+Simple text plugin tests
+"""
+from django.test import TestCase
+from django.test.client import RequestFactory
+
+from cms.api import add_plugin
+from cms.models import Placeholder
+from cms.plugin_rendering import ContentRenderer
+
+from richie.plugins.simple_text_ckeditor.cms_plugins import CKEditorPlugin
+from richie.plugins.simple_text_ckeditor.factories import SimpleTextFactory
+
+
+class SimpleTextTests(TestCase):
+    """SimpleText plugin tests case"""
+
+    def test_simpletext_create_success(self):
+        """
+        Simple text plugin creation success
+        """
+        simpletext = SimpleTextFactory(body="Foo")
+        self.assertEqual("Foo", simpletext.body)
```

Comment:
You should test with a more complex string with accepted html and forbidden tags to test your sanitizer.